docs/network/architecture.md

```diff
@@ -16,6 +16,7 @@ These components are instances of binaries in the [Octez software suite](https:/
 
 Etherlink relies on a sequencer to publish transactions.
 The sequencer receives transactions from EVM nodes, puts them in order, and packages them into an Etherlink block.
+If the Tezos [Data Availability Layer](https://docs.tezos.com/architecture/data-availability-layer) (DAL) is active, the sequencer publishes the block via the DAL; otherwise, it publishes the block directly to Tezos layer 1 via the Smart Rollup outbox.
```

Comment:
the sequencer *can* publish. It does not have to.

And the current approach is to wait (1) for DAL to be ready and running (not only active), and (2) there is a need for it (because this increases the average time for a L2 block to be consider final).

Reply:
Changed to:

> If the Tezos [Data Availability Layer](https://docs.tezos.com/architecture/data-availability-layer) (DAL) is active, the sequencer has the option to publish the block via the DAL if capacity requires it; otherwise, it publishes the block directly to Tezos layer 1.